src/BlazorWebView/src/Maui/build/Microsoft.AspNetCore.Components.WebView.Maui.targets

```diff
@@ -12,6 +12,20 @@
     <RazorComponent Include="**\*.razor" Exclude="$(DefaultItemExcludes);$(DefaultExcludesInProjectFolder)" />
   </ItemGroup>
 
+  <ItemGroup>
+    <!--
+      Web assets (js, css, etc under wwwroot) and .razor files are represented as Content items in the Razor SDK, but run foul of
+      checks in the Android and iOS SDK. This annotates these items with metadata that prevents them from being bundled / examined.
+    -->
+    <Content Update="@(Content->WithMetadataValue('WebContentItem', 'true'))"
+      ExcludeFromContentCheck="true"
+      PublishFolderType="None" />
+
+    <Content Update="@(Content->WithMetadataValue('Extension', '.razor'))"
+      ExcludeFromContentCheck="true"
+      PublishFolderType="None" />
+  </ItemGroup>
+
```

Comment:
Is it required that we do this at project evaluation time? 

It feels that we are doing changes in two places (Razor SDK and here) where we could potentially just have the Maui specific fixes in one place (here) if we were to leverage a target.

Reply:
@javiercn is your suggestion to glob the content added by Web / Razor SDK here?